src/GitHub.Exports/Services/VSServices.cs

```diff
@@ -147,36 +147,48 @@ static string PokeTheRegistryForLocalClonePath()
         const string MRUKeyPath = "MRUSettingsLocalProjectLocationEntries";
         public string SetDefaultProjectPath(string path)
         {
-            string old;
-            using (var newProjectKey = Registry.CurrentUser.OpenSubKey(NewProjectDialogKeyPath, true))
+            var old = String.Empty;
+            try
             {
-                using (var mruKey = newProjectKey?.OpenSubKey(MRUKeyPath, true))
+                var newProjectKey = Registry.CurrentUser.OpenSubKey(NewProjectDialogKeyPath, true);
+                if (newProjectKey == null)
+                    newProjectKey = Registry.CurrentUser.CreateSubKey(NewProjectDialogKeyPath);
+
+                using (newProjectKey)
                 {
+                    var mruKey = newProjectKey.OpenSubKey(MRUKeyPath, true);
```

Comment:
It can't be null, Create will either return something or throw due to permissions or whatever else fails there, which will land in the exception handler.